README.md

```diff
@@ -79,43 +81,69 @@ function AwesomeTitle(text) {
 }
 ```
 
-### Lifecycle
+To diff two virtual nodes and update the DOM, use the `patch` function:
+
+```js
+const element = patch(
+  parent,  // HTML container, e.g., document.body
+  oldNode, // the old VNode
+  newNode  // the new VNode
+)
+```
+
+The `patch` function returns the patched child element.
+
+## Lifecycle Events
+
+Picodom supports element-level life-cycle events and keyed updates, and reserves the following JSX attributes:
+
+#### `key`
```

Comment:
@mindplay-dk `key` is not a lifecycle event though. Can you put this somewhere else?

Reply:
But it's part of the life-cycle control, right?

It has no use other than to control the life-cycle of the element, I think?

So maybe the title of that section should be "Life-cycle Control", not "Life-cycle Events"?

Or maybe just "Reserved Attributes"?

Reply:
Yeah, reserved attributes sounds 👍